.github/gitleaks.toml

```diff
@@ -0,0 +1,25 @@
+
```

Comment:
See comments from this pull request:
https://github.com/arbitraryexecution/automation-testing/pull/92

Reply:
addressed in e1250bd6ccf32489638864a367c380f8b73589cc